.changeset/were-only-dreaming.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Added "out-of-the-box" alpha-milestone search features to scaffolded Backstage apps.
```

Comment:
This needs a more thorough guide for how to migrate existing applications, see other entries in https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md

Reply:
Updated.